packages/rum-recorder/src/boot/startRecording.ts

```diff
@@ -25,10 +25,11 @@ export function startRecording(
     addRecord({ ...rawRecord, timestamp: Date.now() })
   }
 
-  const { stop: stopRecording, takeFullSnapshot } = record({
+  const { stop: stopRecording, takeFullSnapshot, flush } = record({
```

Comment:
what about making explicit the fact the we are flushing mutations?
to avoid any confusions with flush of segment